apps/demo/pages/containers.tsx

```diff
@@ -11,116 +12,264 @@ import {
-          </Card>
-        </Columns>
-      </Section>
-      <Section>
-        <Header>Columns</Header>
-        <Columns columns={6}>
-          {[...Array(12)].map((_, i) => (
-            <Card
-              key={i}
-              appearance="outlined"
-              className="skc-headline-large !grid h-20 place-content-center"
-            >
-              {i + 1}
-            </Card>
-          ))}
-        </Columns>
-      </Section>
+const CardSection: FC = () => (
+  <Section>
+    <Header>Card</Header>
+    <Columns columns={3}>
+      <Card appearance="elevated">
+        <CardHeader
+          icon={<MaterialIcon icon="person" />}
+          title="Atipol Sukrisadanon"
+          subtitle="Foreign Languages teacher"
+        />
+        <Image
+          src="/images/atipol.jpg"
+          width={800}
+          height={533}
+          alt="Atipol Sukrisadanon"
+          priority
+          className="w-full"
+        />
+        <CardContent>
+          <p>
+            Atipol Sukrisadanon is arguably one of the teachers in Suankularb
+            Wittayalai School.
+          </p>
+          <Actions>
+            <Button appearance="filled">Learn more</Button>
+          </Actions>
+        </CardContent>
+      </Card>
+    </Columns>
+  </Section>
+);
+
+const ColumnsSection: FC = () => (
+  <Section>
+    <Header>Columns</Header>
+    <Columns columns={6}>
+      {[...Array(12)].map((_, i) => (
+        <Card
+          key={i}
+          appearance="outlined"
+          className="skc-headline-large !grid h-20 place-content-center"
+        >
+          {i + 1}
+        </Card>
+      ))}
+    </Columns>
+  </Section>
+);
+
+const DialogSection: FC = () => {
+  const [showRemStudents, setShowRemStudents] = useState<boolean>(false);
+  const [showReport, setShowReport] = useState<boolean>(false);
+  const [view, setView] = useState<"bug-report" | "feature-request">(
+    "bug-report"
+  );
+
+  return (
+    <Section>
+      <Header>Dialog</Header>
+
+      {/* Dialog triggers */}
+      <Actions align="left">
+        <Button
+          appearance="filled"
+          icon={<MaterialIcon icon="delete" />}
+          dangerous
+          onClick={() => setShowRemStudents(true)}
+        >
+          Remove students
+        </Button>
+        <Button
+          appearance="outlined"
+          icon={<MaterialIcon icon="bug_report" />}
+          onClick={() => setShowReport(true)}
+        >
+          Report issue
+        </Button>
+      </Actions>
+
+      {/* Remove students Dialog */}
+      <Dialog open={showRemStudents} onClose={() => setShowRemStudents(false)}>
+        <DialogHeader
+          title="Remove students?"
+          desc="The following students will no longer have access to the
+            organization “Kornor.”"
+        />
+        <DialogContent>
+          <List divided>
+            <ListItem align="center" lines={1}>
+              <ListItemContent title="Sadudee Theparree" />
```

Comment:
😳

Reply:
this was literally in the Specs
did you not see it

Reply:
Yes